website/views.py

```diff
@@ -191,23 +192,27 @@ def get_dashboard_nodes(auth):
     """
     user = auth.user
 
-    contributed = user.node__contributed  # nodes user contributed to
-
-    nodes = contributed.find(
-        Q('category', 'eq', 'project') &
-        Q('is_deleted', 'eq', False) &
-        Q('is_registration', 'eq', False) &
-        Q('is_folder', 'eq', False)
+    nodes = Node.find_for_user(
+        user,
+        subquery=(
+            Q('category', 'eq', 'project') &
+            Q('is_deleted', 'eq', False) &
+            Q('is_registration', 'eq', False) &
+            Q('is_folder', 'eq', False)
+        )
     )
 
     if request.args.get('no_components') not in [True, 'true', 'True', '1', 1]:
-        comps = contributed.find(
-            # components only
-            Q('category', 'ne', 'project') &
-            # exclude deleted nodes
-            Q('is_deleted', 'eq', False) &
-            # exclude registrations
-            Q('is_registration', 'eq', False)
+        comps = Node.find_for_user(  # NOTE - this used to be a find on nodes above. Does this mess it up?
+            user,
+            (
+                # components only
+                Q('category', 'ne', 'project') &
```

Comment:
See above about the difference between projects and components in the modern world.